### PR TITLE
Add ARB, OP, MATIC & DASH to Ledger, Add DASH to Trezor

### PR DESCRIPTION
**Pull request by GiMa-Maya**

### CODE REVIEW

File: packages/wallets/ledger/src/clients/dash.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import { getWalletFormatFor } from "../helpers/derivationPath.ts";
+import { UTXOLedgerInterface } from "../interfaces/LedgerInterfaces.ts";
+
+export class DashLedger extends UTXOLedgerInterface {
+  constructor(derivationPath: DerivationPathArray = NetworkDerivationPath.DASH) {
+    super();
+    this.addressNetwork = getNetwork(Chain.Dash);
+    this.chain = "dash";
+    this.derivationPath = derivationPathToString(derivationPath);
+    this.walletFormat = getWalletFormatFor(this.derivationPath) as "legacy" | "bech32" | "p2sh";
```
Comment (GiMa-Maya) [inline packages/wallets/ledger/src/clients/dash.ts:15]: Double checking this with Dash team

**Comment by towanTG** [comment]
🔥

**Comment by chillios-dev** [comment]
@GiMa-Maya running lint and build - when green will merge ;)

**Comment by chillios-dev** [comment]
@GiMa-Maya I'll need to do some changes in your pr - will checkout and continue on my own ^^

**Comment by GiMa-Maya** [comment]
@chillios-dev 

Alright. Let me know if I can help!

**Comment by chillios-dev** [comment]
continued in #677

**Pull request closed without merging**